app/clients/freshdesk.py

```diff
@@ -104,7 +104,8 @@ def send_ticket(self) -> int:
                     timeout=5,
                 )
                 response.raise_for_status()
-
+                if response.status_code == 201:
```

Comment:
I wonder if a scenario could happen where we wouldn't receive a `201` and skip this log?